bosonprotocol/core-components
# feat(core-sdk): add voidOffer method

## Description

This adds a top-level core-sdk method to void an offer.

## How to test
```js
const txResponse = await coreSDK.voidOffer(offerId)
```


---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1201954774090265
